scipy/stats/_hypotests.py

```diff
@@ -692,23 +692,25 @@ def barnard_exact(table, alternative="two-sided", pooled=True, num_it=3):
         * 'greater': one-sided
 
     pooled : bool, optional
-        Whether to compute score statistic with pooled variance or
-        unpooled (Wald statistic)
+        Whether to compute score statistic with pooled variance (Student's
+        t-test) or unpooled variance (Welch's t-test). Default is ``True`` :
+        Statistic test is computed using pooled variance
 
     num_it : int, optional
         Number of iterations of the grid search. Default is 3. Must be
-        non-negative.
+        non-negative. In most cases, 3 iterations is perfectly enough to
+        reach a precision a goood precision. Above a certain number (around
```

Comment:
```suggestion
        reach a good precision. Above a certain number (around
```

Reply:
Update in 7707d9893268328edb4bd2001a96edf6840c9243